core/src/main/java/com/linecorp/armeria/server/AnnotatedHttpServiceConfigurator.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableList.Builder;
+
+import com.linecorp.armeria.internal.annotation.AnnotatedHttpService;
+import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction;
+import com.linecorp.armeria.server.annotation.RequestConverterFunction;
+import com.linecorp.armeria.server.annotation.ResponseConverterFunction;
+
+/**
+ * A configurator to configure {@link ExceptionHandlerFunction}s, {@link RequestConverterFunction}s
+ * or {@link ResponseConverterFunction}s for an {@link AnnotatedHttpService}.
+ */
+final class AnnotatedHttpServiceConfigurator {
```

Comment:
- How about renaming to `AnnotatedHttpServiceExtensions`? It does not configure anything anymore but just provides the extensions.
- How about building the 3 lists in `ofExceptionHandlersAndConvertes()` so that you do not need to call `ImmutableList.Builder.builder()` in the getters?

Reply:
> How about renaming to AnnotatedHttpServiceExtensions? It does not configure anything anymore but just provides the extensions.

Good idea! :)

> How about building the 3 lists in ofExceptionHandlersAndConvertes() so that you do not need to call ImmutableList.Builder.builder() in the getters?

There was some misunderstanding during the review. I will fix it.